simplesat/transaction.py

```diff
@@ -29,16 +30,20 @@ class RemoveOperation(Operation):
 class Transaction(object):
 
     def __init__(self, pool, decisions, installed_map):
-        self.operations = []
-
+        self.operations = self._safe_operations(pool, decisions, installed_map)
+        self.pretty_operations = []
```

Comment:
The other code is still here for easy comparison. If we decide to do things the new way it should be removed.